### DISCUSSION TITLE
Observations endpoint only has last week of data?

### DISCUSSION THREAD

**Discussion by sbrowngigster** [weather-gov/api#646] [General]
I'm trying to get temperature and precipitation across several stations for the last year. I'm getting Response 200 with empty feature list for any time period further back than one week. For instance, this returns empty feature list:
`https://api.weather.gov/stations/KLAF/observations?start=2023-07-11T00:00:00Z&end=2023-07-12T00:00:00Z`
While this returns the data:
`https://api.weather.gov/stations/KLAF/observations?start=2023-07-19T00:00:00Z&end=2023-07-20T00:00:00Z`
I saw the same thing across dozens of stations I tried. I'm trying to figure out if the problem is in on my end or the API itself.

Thanks

**Comment by JJKraw** [comment]
See https://github.com/weather-gov/api/discussions/275

The API is not meant to be a repository for historical data.

**Reply by sbrowngigster** [reply]
Understood, thanks for taking the time to clarify.
